docs/miscellaneous.rst

```diff
@@ -58,6 +61,21 @@ So for the following contract snippet::
 
 The position of ``data[4][9].b`` is at ``keccak256(uint256(9) . keccak256(uint256(4) . uint256(1))) + 1``.
 
+.. _bytes-and-string:
+
+``bytes`` and ``string``
+------------------------
+
+``bytes`` and ``string`` are encoded identically. For short byte arrays, they store their data in the same
+slot where also the length is stored. In particular: if the data is at most ``31`` bytes long, it is stored
+in the higher-order bytes (left aligned) and the lowest-order byte stores ``length * 2``.
+For long byte arrays, the main slot stores ``length * 2 + 1`` and the data is stored as usual in ``keccak256(slot)``.
+This means that a short array can be distinguished from a long array by checking if the lowest bit is set:
```

Comment:
@bit-shift I think you can remove passive voice here and it is still OK, so:

"This means that you can distinguish a short array from a long array by checking if the lowest bit is set."